Assets/Scripts/Fire.cs

```diff
@@ -0,0 +1,50 @@
+﻿    using System.Collections;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+
+public class Fire : MonoBehaviour
+{
+    public float scaleFire;
+    public float speedDecreacing;
+    public float minScale;
+    public int densityFire;
+    private ParticleSystem part;
+    private ParticleSystem.ShapeModule sh;
+    private ParticleSystem.EmissionModule em;
+
+    // Start is called before the first frame update
```

Comment:
You can remove these default comments if you want, they don't really contribute to much. It is annoying they appear every time, I already tried multiple times to replace the default script template but to no avail haha
```suggestion
```